packages/protocol/test/stability/exchange.ts

```diff
@@ -548,30 +1095,30 @@ contract('Exchange', (accounts: string[]) => {
         })
 
         describe('when the oldest oracle report is not expired', () => {
-          const expectedStableAmount = getBuyTokenAmount(
-            goldTokenAmount,
+          const expectedGoldAmountNoReportExpired = getSellTokenAmount(
+            buyStableAmount,
             updatedGoldBucket,
             updatedStableBucket
           )
 
-          it('the exchange should succeed', async () => {
-            await exchange.exchange(
-              goldTokenAmount,
-              expectedStableAmount.integerValue(BigNumber.ROUND_FLOOR),
-              true,
+          it(`the exchange should succeed`, async () => {
```

Comment:
nit: was the change of quote style intentional? Nothing's being interpolated into this string...